### DISCUSSION TITLE
Github Actions - Deployment Protection Rules - Avoid multiple approvals

### DISCUSSION THREAD

**Discussion by LegendAlec-M** [community/community#102095] [Actions]
### Select Topic Area

Question

### Body

Github Actions: On a workflow with multiple jobs, and with a "deployment protection rule", at the moment each job needs a separate approval. It causes issues when the workflow deploys to 2 different e.g. api backend and js frontend. So if by mistake only the api backend is approved, and the js frontend is not, the application breaks.

Is there a way that the "deployment protection rule" can be approved at the entire workflow level and all jobs inside the workflow run without approval. 

Thanks in advance
Al

**Comment by yashmalvade2002** [comment]
res

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @alecmadeira-itech ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by LegendAlec-M** [reply]
Cheer @queenofcorgis, thank you.

**Comment by abower-digimarc** [comment]
I don't have an answer, I have the same problem. We'd like to be able to 'link' or 'bundle' multiple jobs under a single approval in a workflow.
